backend/src/apiserver/visualization/roc_curve.py

```diff
@@ -34,7 +34,7 @@
 # trueclass
 # true_score_column
 
-if "is_generated" is not in variables or variables["is_generated"] is False:
+if variables.get("is_generated", False) is False:
```

Comment:
```
if not variable.get("is_generated", False): 
```

Reply:
That changes the meaning of the code but I am good with it if you are.